export const FailProvider = async () => {
  throw new Error('Provider failed');
};
